test/e2e/generated/bindata.go

```diff
@@ -5,9 +5,9 @@
 // examples/aws_ebs/aws-ebs-web.yaml
```

Comment:
Ditto for other auto-gen files in this commit. This commit should only contain manual edits. Read the comments at the top of the files to figure out if they should be manually edited or auto-generated.